emails/file-expiry.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+    Hr,
+    Link,
+    Section,
+    Text
+} from "@react-email/components";
+import { z } from "zod";
+import { WATcloudEmail } from "./_common/watcloud-email";
+
+const EmailProps = z.object({
+    name: z.string(),
+    paths: z.array(z.string()),
+    daysForExpiry: z.string(),
+    deletionDate: z.string()
+});
+
+type EmailProps = z.infer<typeof EmailProps>;
+
+const Email = (props: EmailProps) => {
+    const { name, paths, daysForExpiry, deletionDate} = EmailProps.parse(props);
+
+    const previewText = `You have expired files in the WATO drives`;
+
+    const expiredPaths = (
+        <div>
```

Comment:
The style of this list seems to not match the surrounding text:

<img width="649" alt="image" src="https://github.com/user-attachments/assets/fc074c47-1401-4436-980c-a39886d8b6bd">

You can reference the onboarding email for list styles: https://github.com/WATonomous/watcloud-emails/blob/a5a9855632c3810d08d8e7f26040ab9758249ecc/emails/onboarding.tsx#L23